services/frontend/src/utils/helperFunctions.js

```diff
@@ -8,6 +8,23 @@ export const arrayToObject = ({processor = identity, property = "id"} = {}) => (
     return acc;
 };
 
+export const sortObjectsByProperty = (objects, property) => (
+    objects.sort((a, b) => a[property].localeCompare(b[property]))
+);
+
+export const parseDateStringToYMD = (dateString) => {
+    if (!dateString) {
+        return "";
+    } else {
+        const epochDate = Date.parse(dateString);
+        const dateObject = new Date(epochDate);
```

Comment:
You can parse the `dateString` directly with the `Date` constructor; no need to convert it to epoch first. i.e.

```suggestion
        const dateObject = new Date(dateString);
```

Reply:
Yep, I removed line 19 and replaced `epochDate` with `dateString`.